common/vm_hot_add_remove_ctrl.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2025 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Hot add or remove VM disk or USB controller using 'vmware_guest_controller' module.
+# Parameters:
+#   vm_controller_state: valid value is 'present' or 'absent'.
+#   vm_controller_type: supported value is 'buslogic', 'lsilogic', 'lsilogicsas', 'paravirtual',
+#     'sata', 'nvme', 'usb2', or 'usb3'.
+#   disk_controller_number: disk controller bus number, valid value is 0, 1, 2, or 3.
+#     Required when 'vm_controller_state' is set to 'absent' and 'vm_controller_type' is not 'usb2'
+#     or 'usb3'.
+#   vm_controller_sleep_time (optional): the sleep time in seconds after VM reconfiguration.
+# Return:
+#   vm_controller_facts
+#
+- name: "Set fact of the list of supported VM controllers in this task"
+  ansible.builtin.set_fact:
+    vm_controller_list:
+      - 'buslogic'
+      - 'lsilogic'
+      - 'lsilogicsas'
+      - 'paravirtual'
+      - 'sata'
+      - 'nvme'
+      - 'usb2'
+      - 'usb3'
+
+- name: "Check required parameters"
+  ansible.builtin.assert:
+    that:
+      - vm_controller_state is defined
+      - vm_controller_state | lower in ['present', 'absent']
+      - vm_controller_type is defined
+      - vm_controller_type | lower in vm_controller_list
```

Comment:
We can limit the value of `vm_controller_state` and `vm_controller_type` to be in lower case, to avoid many `lower` filters. And the vars passed from `include_tasks` are already in lower case.